wordpress/content.md

```diff
@@ -31,6 +31,10 @@ If you'd like to use an external database instead of a linked `mysql` container,
 	docker run --name some-wordpress -e WORDPRESS_DB_HOST=10.1.2.3:3306 \
 	    -e WORDPRESS_DB_USER=... -e WORDPRESS_DB_PASSWORD=... -d wordpress
 
+To get wordpress working with nginx, will require a custom build of nginx to enable fpm support. You'll also need to expose port 9000 (default fpm port for the wordpress build):
```

Comment:
"require a custom build of nginx to enable fpm support" has me a little confused.  I thought FastCGI support in nginx was well-baked and it was just a matter of configuring it appropriately?

@md5 you got this connection working, didn't you? (`nginx` + `wordpress:fpm`)